File: javascript/raindrops/raindrops.js
export const convert = (n) => {
  let s = '';

  if (n % 3 === 0) s += 'Pling';
  if (n % 5 === 0) s += 'Plang';
  if (n % 7 === 0) s += 'Plong';

  return s || n.toString();
};

File: csharp/raindrops/Raindrops.cs
using System.Text;

public static class Raindrops
{
    public static string Convert(int number)
    {
        var str = new StringBuilder(15);

        if (number % 3 == 0) str.Append("Pling");
        if (number % 5 == 0) str.Append("Plang");
        if (number % 7 == 0) str.Append("Plong");

        return str.Length > 0 ? str.ToString() : number.ToString();
    }
}
